content/cli/v7/commands/npm-unpublish.md

```diff
@@ -17,7 +17,7 @@ github_path: docs/content/commands/npm-unpublish.md
 ### Synopsis
 
 To learn more about how the npm registry treats unpublish, see our <a
-href="https://www.npmjs.com/policies/unpublish" target="_blank"
+href="https://docs.npmjs.com/policies/unpublish" target="_blank"
```

Comment:
This needs to be updated in https://github.com/npm/cli/blob/latest/docs/content/commands/npm-unpublish.md